fiwareskuld/phase/phase0_generateuserlist_alt.py

```diff
@@ -85,7 +85,7 @@ def get_yellow_red_users(self):
                 continue
             if remaining < 0:
                 red_users.append(user)
-            elif remaining <= settings.NOTIFY_BEFORE_EXPIRED:
+            elif remaining <= settings.NOTIFY_BEFORE_TRIAL_EXPIRED:
```

Comment:
I have a question here, if I am using this script both Trial and Community users, how can we see the NOTIFY_BEFORE_COMMUNITY_EXPIRED too?